.github/workflows/documentation.yml

```diff
@@ -0,0 +1,66 @@
+name: Build documentation
+
+on:
+  schedule:
+    - cron: '0 0 * * SAT' # every Saturday
+
+jobs:
+  update-docs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout blog
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.USER_TOKEN }}
+
+      - name: Install OCaml
+        uses: avsm/setup-ocaml@v1
+        with:
+          ocaml-version: 4.09.1
+
+      - name: Pin OASIS
+        run: opam pin add oasis https://github.com/BinaryAnalysisPlatform/oasis.git
+
+      - name: Checkout bap
+        uses: actions/checkout@v2
+        with:
+          repository: BinaryAnalysisPlatform/bap
+          path: bap.master
```

Comment:
What is the purpose of checking out into a separate folder?

Reply:
because `checkout` rewrites the current directory, i.e. all the blog files.